packages/blade/src/components/Card/CardContext.tsx

```diff
@@ -11,9 +12,29 @@ const useVerifyInsideCard = (componentName: string): CardContextType => {
   return true;
 };
 
+/**
+ * Verify if the passed childrens are only of allowedComponents list
+ */
+const useVerifyAllowedComponents = (
+  children: React.ReactNode,
+  componentName: string,
+  allowedComponents: string[],
+): void => {
+  React.Children.forEach(children, (child) => {
```

Comment:
We're entering into lot of runtime checks territory. Similar to https://github.com/razorpay/blade/issues/272 I think we should start considering moving all runtime checks to development mode only /cc @kamleshchandnani